src/config.rs

```diff
@@ -6,8 +7,11 @@ use std::io::IsTerminal;
 use std::path::Path;
 use std::path::PathBuf;
 
+use crate::dir_walker::Operater;
 use crate::display::get_number_format;
 
+pub static DAY_SECEONDS: i64 = 24 * 60 * 60;
```

Comment:
typo!

Reply:
Sorry, due to my carelessness, I did not correct this spelling error in time. I will resubmit a PR for processing.

Reply:
Not a problem! we all do it.